enclave-runtime/src/lib.rs

```diff
@@ -69,10 +70,18 @@ use itp_stf_state_handler::{
 };
 use itp_storage::StorageProof;
 use itp_types::{Block, Header, SignedBlock};
-use its_sidechain::top_pool_rpc_author::{
-	global_author_container::GlobalAuthorContainer, traits::GetAuthor,
+use its_sidechain::{
+	aura::block_importer::{BlockImport, BlockImporter},
+	primitives::{
+		traits::SignedBlock as SignedSidechainBlockT,
+		types::block::SignedBlock as SignedSidechainBlock,
+	},
+	state::SidechainDB,
+	top_pool_executor::TopPoolOperationHandler,
+	top_pool_rpc_author::{global_author_container::GlobalAuthorContainer, traits::GetAuthor},
```

Comment:
I know it's not obvious from the naming, but I tried to put everything sidechain related in the `top_pool_execution.rs`, to clearly separate it. IMO we should not add more code to this `lib.rs` unless really necessary. It might make sense to rename the `top_pool_execution.rs` to something with `sidechain`.